ecosystem/web-wallet/src/core/components/BoringAvatar.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+interface BoringAvatarProps {
+  type?: 'beam' | 'marble' | 'bauhaus'
+}
+
+/**
+ * @see https://boringavatars.com/
+ */
+export function GraceHopperBoringAvatar({
+  type = 'beam',
+}: BoringAvatarProps) {
+  switch (type) {
+    case 'marble': {
+      return (
+        <svg viewBox="0 0 80 80" fill="none" role="img" xmlns="http://www.w3.org/2000/svg" width="100%" height="100%">
```

Comment:
Might want to run svgo on these. https://github.com/svg/svgo

Reply:
Yeah its a good thought, though they do have an npm library 
```bash
npm install boring-avatars
```

but I just wanted to use one of them so I'll just keep it the way it is. If we really start using more of them then ill look into this